Autoescuela/Autoescuela/AppDelegate.swift

```diff
@@ -7,31 +7,24 @@
 //
 
 import UIKit
+import Foundation
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
-
+    var window: UIWindow?
+    
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+       
+      window = UIWindow(frame: UIScreen.main.bounds)
+            let testVC = ViewController(nibName: nil, bundle: nil)
```

Comment:
Change the name of the variable into mainViewController, testVC isn't a good name to have in the appDelegate